evoting/service/service.go

```diff
@@ -174,6 +179,47 @@ func (s *Service) Login(req *evoting.Login) (*evoting.LoginReply, error) {
 	return &evoting.LoginReply{Token: token, Admin: admin, Elections: elections}, nil
 }
 
+// LookupSciper calls https://people.epfl.ch/cgi-bin/people/vCard?id=sciper
+// to convert scipers to names
+func (s *Service) LookupSciper(req *evoting.LookupSciper) (*evoting.LookupSciperReply, error) {
+	url := fmt.Sprintf("https://people.epfl.ch/cgi-bin/people/vCard?id=%s", req.Sciper)
+	resp, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	reply := &evoting.LookupSciperReply{}
+	search := regexp.MustCompile("[:;]")
+	for _, line := range strings.Split(string(body), "\n") {
+		fr := search.Split(line, 2)
+		if len(fr) != 2 {
+			continue
+		}
+		value := strings.Replace(fr[1], "CHARSET=UTF-8:", "", 1)
+		switch fr[0] {
+		case "FN":
+			reply.FullName = value
+		case "EMAIL":
+			reply.Email = value
+		case "TITLE":
+			reply.Title = value
+		case "URL":
+			reply.URL = value
+		}
+	}
+	if reply.FullName == "" {
+		return nil, errors.New("didn't find sciper")
+	}
+
+	log.Lvl2("Got vcard: %v", reply)
```

Comment:
The default log level for product servers is now 2; I think logging precise data other than "this connection from this remote addr" should be at level 3.